### ISSUE TITLE
"Illegal instruction: 4" and crash on OSX 10.7

### ISSUE THREAD

**Issue by starless72** [tatsuhiro-t/aria2#600]
Hi, I downloaded release 1.21.0 for Mac (aria2-1.21.0-osx-darwin.dmg) and installed it on OSX 10.7.4, but it is not able to start at all. Whatever argument I use (even just "aria2c -v") I get the following message:

`Illegal instruction: 4`

I attach the related lines from Console log and the bash crash report mentioned therein.

[Illegal_instruction_4_Console.txt](https://github.com/tatsuhiro-t/aria2/files/179640/Illegal_instruction_4_Console.txt)
[bash_2016-03-18-125258_VirtualOSX_crash.txt](https://github.com/tatsuhiro-t/aria2/files/179641/bash_2016-03-18-125258_VirtualOSX_crash.txt)

**Comment by starless72** [comment]
For the record, I'm trying the new version of aria2 because I experienced problems with the previous 1.19.3, too, on the same OSX version 10.7.4: I got the following message whenever I tried to download an HTTPS URL (e.g. https://www.google.com):

`Segmentation fault: 11`

I attach again Console lines and crash report.

[Segmentation_fault_11_Console.txt](https://github.com/tatsuhiro-t/aria2/files/179677/Segmentation_fault_11_Console.txt)
[aria2c_2016-03-18-130020_VirtualOSX_crash.txt](https://github.com/tatsuhiro-t/aria2/files/179678/aria2c_2016-03-18-130020_VirtualOSX_crash.txt)

This is similar to #275, but I'm not using any proxy, and that issue is rather old and should be fixed by now.
Anyway, the new version made things even worse, because aria2c cannot start at all now.

Is there a minimum supported version of OSX?
